<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>701</width>
     <height>331</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>название сорта</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>степень обжарки</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>молотый/в зернах</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> описание вкуса</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>цена</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>объем упаковки</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
